apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -176,8 +176,8 @@
 
             <a class="dropdown-item" href="<%= instructorLink %>" id="navbar-user-view-authn-instructor">
               <div class="d-flex flex-row justify-content-left">
-                <div class="col-1 px-0"><% if (authnViewTypeMenuChecked == 'instructor') { %>&check;<% } %></div>
-                <div class="col pl-0">
+                <span class="<% if (authnViewTypeMenuChecked != 'instructor') { %>invisible<% } %>">&check;</span>
```

Comment:
Nit:
```suggestion
                <span class="<% if (authnViewTypeMenuChecked !== 'instructor') { %>invisible<% } %>">&check;</span>
```

same for the other checks.

Reply:
We can do this when this all gets converted to TS and gets checked by ESLint.